packages/libs/react-ui/src/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import { Button, SystemIcon } from './../../';
+import {
+  INotificationProps,
+  Notification,
+  NotificationFooter,
+  NotificationHeader,
+} from './Notification';
+import { colorVariants } from './Notification.css';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    selectIcon: keyof typeof SystemIcon;
+    text: string;
+  } & INotificationProps
+> = {
+  title: 'Layout/Notification',
+  argTypes: {
+    selectIcon: {
+      options: Object.keys(SystemIcon) as (keyof typeof SystemIcon)[],
+      control: {
+        type: 'select',
+      },
+    },
+    title: {
+      control: {
+        type: 'text',
+      },
+    },
+    simple: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    color: {
+      options: [
+        'default',
+        ...(Object.keys(colorVariants) as (keyof typeof colorVariants)[]),
+      ],
+      control: {
+        type: 'select',
+      },
+    },
+    expand: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    displayCloseButton: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<
+  {
+    selectIcon: keyof typeof SystemIcon;
+  } & INotificationProps
+>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/7.0/react/api/csf
+ * to learn how to use render functions.
+ */
+
+export const Primary: Story = {
+  name: 'Notification',
+  args: {
+    selectIcon: 'Information',
+    title: 'Notification title',
+    displayCloseButton: true,
+    expand: false,
+    color: undefined,
+  },
+  render: ({
+    selectIcon,
+    title,
+    displayCloseButton: displayCloseButton,
+    expand,
+    color,
+    simple,
```

Comment:
Let's try to come up with naming convention for booleans so that it is clear. I was thinking the "comparative" form of the word: so similar to `disabled`

In this case it would be "expanded" and "simplified". What do you think?